sklearnserver/README.md

```diff
@@ -0,0 +1,40 @@
+## Testing instructions
+
+From the [upstream usage example](https://kserve.github.io/website/master/modelserving/v1beta1/sklearn/v2/#deploy-the-model-with-rest-endpoint-through-inferenceservice/), this rock can be tested locally using:
+
+Launch the server with:
+```
+# download the model locally
+mkdir sample_model
+gsutil cp -r gs://kfserving-examples/models/sklearn/1.0/model ./sample_model/
```

Comment:
Perhaps we need a small pre-requisites section with this, docker, and any other sw that is used for testing OR provide a `requirements.txt` file that people can just use. Up to you, either works for me.

Reply:
Added a prereqs section - wdyt?